main.go

```diff
@@ -28,12 +28,21 @@ func (s Service) AddressAndPort() string {
 
 var timeout = flag.Int64("timeout", 60, "time to wait for all services to be up (seconds)")
 var httpPort = flag.Int("httpport", 0, "wait for an http request if target port is given port")
-var ignorePort = flag.Int("ignoreport", 0, "don't wait for services on this port to be up")
+var _ignorePort = flag.String("ignoreport", "", "don't wait for services on these ports to be up (separated by comma)")
 
 func main() {
 	setupUsage()
 	flag.Parse()
 
+	var ignorePortStr = strings.Split(*_ignorePort, ",")
+	var ignorePorts []int
+
+	for _,portStr := range ignorePortStr {
+	  	if s, err := strconv.ParseInt(portStr, 10, 64); err == nil {
```

Comment:
[`strconv.Atoi`](https://golang.org/pkg/strconv/#Atoi) does the same, just a bit shorter. It also converts directly to `int`, so there's no need to convert the variable to an int below.